Summary/SOP_Review.md

```
@@ -0,0 +1,534 @@
+      "args": ["run", "--extra", "mcp", "python", "-m", "asp.mcp.server"],
+      "env": {
+        "ANTHROPIC_API_KEY": "${ANTHROPIC_API_KEY}",
+        "ASP_LOG_LEVEL": "INFO"
+      }
+    }
+  }
+}
+```
+
+**Starting MCP Server:**
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)